DabTester.py

```diff
@@ -11,19 +11,20 @@ def __init__(self,broker):
         self.dab_client.connect(broker,1883)
         self.verbose = False
 
-    def execute_cmd(self,device_id,dab_dab_request_body_topic,dab_request_body="{}"):
-        self.dab_client.request(device_id,dab_dab_request_body_topic,dab_request_body)
+    def execute_cmd(self,device_id,dab_request_topic,dab_request_body="{}"):
+        self.dab_client.request(device_id,dab_request_topic,dab_request_body)
         if self.dab_client.last_error_code() == 200:
             return 0
         else:
             return 1
     
     def Execute_Test_Case(self, device_id, test_case):
-        (dab_dab_request_body_topic, dab_request_body, validate_output_function, expected_response_code)=test_case
-        test_result = TestResult(device_id, dab_dab_request_body_topic, dab_request_body, "UNKNOWN", "", [])
-        print("\ntesting", dab_dab_request_body_topic, " ", dab_request_body, "... ", end='', flush=True)
+        (dab_request_topic, dab_request_body, validate_output_function, expected_response_code)=test_case
+        test_result = TestResult(device_id, dab_request_topic, dab_request_body, "UNKNOWN", "", [])
+        print("\ntesting", dab_request_topic, " ", dab_request_body, "... ", end='', flush=True)
         start = datetime.datetime.now()
-        if self.execute_cmd(device_id, dab_dab_request_body_topic, dab_request_body) == 0:
+        test_result.response = self.dab_client.response()
```

Comment:
Collecting response before execution of the test?

Reply:
Yea, it was error out.

Reply:
Not clear. Wondering how will the response be set before execute_cmd.